Search2DMatrix.cpp

```diff
@@ -0,0 +1,41 @@
+// Approach 1 - Apply binary search twice(once to find the row and again to find the column ion that row.)
+// Time Complexity - O(log n). Sine we are using the Binary search to find the row to find the element and Binary search again to find the column at that row again.
+// Space Comlexity - O(n). Since it depends on the number of elements in the 2D array? Is my answer correct?
```

Comment:
Here Space complexity is O(1) as we are using any extra space for searching an element.